src/screens/WalletsScreen/index.tsx

```diff
@@ -484,12 +493,17 @@ class WalletsComponent extends React.Component<Props, WalletsState> {
         const { wallets } = this.props;
         const { tab } = this.state;
         const depositTab = { label: this.renderTabs()[0].label, index: 0 };
+        const withdrawTab = { label: this.renderTabs()[1].label, index: 1 };
 
         if (tab !== depositTab.label && value.type !== 'fiat') {
             this.onTabChange(depositTab.index, depositTab.label);
             this.onCurrentTabChange(depositTab.index);
         }
 
+        if (tab === withdrawTab.label) {
```

Comment:
```suggestion
        if (tab === withdrawTab.label && value) {
```

Reply:
in walletSelectionChange value can't be undefined
![image](https://user-images.githubusercontent.com/39953699/105357149-101b2500-5bfd-11eb-846c-b17b0d923801.png)